AllReadyApp/Web-App/AllReady/Areas/AreaNameConstants.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace AllReady.Areas
+{
+    public static class AreaNameConstants
+    {
+        public const string ADMIN_AREA_NAME = "Admin";
```

Comment:
We haven't been very consistent with this but I think we generally have been using Pascal casing for public constants. In that case, this would be `AdminAreaName` instead of `ADMIN_AREA_NAME`. 

This PR is fine as is but I thought I would mention it as something for the future. Maybe we can add a short code style document to our repo, or link to the one the .NET team uses. It would be good to be consistent in the future.